src/configuration/Options.cpp

```diff
@@ -48,7 +48,7 @@ void Options::initializeDefaultValues()
     _boolOptions[DUMP_BOUNDS] = false;
     _boolOptions[SOLVE_WITH_MILP] = false;
     _boolOptions[PERFORM_LP_TIGHTENING_AFTER_SPLIT] = false;
-    _boolOptions[PARALLEL_DEEPSOI] = true;
```

Comment:
I think "positive" flags are easier to understand than negative ones, due to double negations. Why was it bothering you the way it was before?

Reply:
Somehow boost option only seems to be able to set the flag to true. I played with it but didn't get it to work the other way..